### PR TITLE
Plans: update copy on the My Plan page

### PR DESCRIPTION
**Pull request by donlair**
#### Changes proposed in this Pull Request

Tweaking copy on My Plan page to increase clarity, including removing a reference to "hundreds" of available themes and changing Business plan storage limit reference to 200GB, from unlimited.

#### Testing instructions

Navigate to http://calypso.localhost:3000/plans/my-plan/SITENAME with a Business plan account and confirm that all cards are properly displayed, and that copy in the updated cards has no typos or grammatical errors.

Here's a screenshot of what it should look like:
screencapture-calypso-localhost-3000-plans-my-plan-donlair-com-2020-03-13-14_28_38

Confirm that

* The appropriate cards, as defined in `/client/blocks/product-purchase-features-list/index.jsx`, are properly displayed

Fixes #

### CODE REVIEW

File: client/blocks/product-purchase-features-list/sell-online-paypal.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default localize( ( { isJetpack, translate } ) => {
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
-				buttonText={ translate( 'Sell online' ) }
+				buttonText={ translate( 'Collect payments' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features-list/sell-online-paypal.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Collect Payments and Donations' )` *ES Score: 7*

**Comment by lancewillett** [comment]
Related, minor — but if we're "in the neighborhood" to fix, let's add it:

#38349

**Comment by donlair** [comment]
@lancewillett I should be able to add that to the tweaks. I poked around other pages and noticed this style (18px in the title card tagline w/ 15px in the subcards), for example /email/[sitename].

Pages like My Home follow that pattern as well, but with different font size pairs.

Should I go ahead and drop the font size in the title card to 15px to match the rest of the page?

**Comment by sixhours** [comment]
While you're at it... :D We're removing use cases of `font-weight: 300` throughout Calypso (see #39596) and I think that subhead is currently using 300.

**Comment by donlair** [comment]
> While you're at it... :D We're removing use cases of `font-weight: 300` throughout Calypso (see #39596) and I think that subhead is currently using 300.

@sixhours  sure, I'll bump that to 400. Should it be 15px?

**Comment by sixhours** [comment]
> @sixhours sure, I'll bump that to 400. Should it be 15px?

I've been going by our Typography docs in devdocs: https://wpcalypso.wordpress.com/devdocs/typography

So I think 14px would be best.

**Comment by lancewillett** [comment]
Thanks for the teamwork here, y'all. Little changes add up to a major lift.

**Comment by donlair** [comment]
I changed the page Headline font to 21 px, and changed the title card body copy to 14px / weight to 400.

**Comment by sixhours** [comment]
This works well for me! 👍

File: client/blocks/product-purchase-features-list/video-audio-posts.jsx [hunk 2]
```diff
@@ -19,8 +19,8 @@ import videoImage from 'assets/images/illustrations/video-hosting.svg';
 function getDescription( plan, translate ) {
 	if ( isWpComBusinessPlan( plan ) ) {
 		return translate(
-			'Enrich your posts and pages with video or audio. Upload as much media as you want, ' +
-				'directly to your site — the Business Plan has unlimited storage.'
+			'Enrich your site and boost engagement with video and audio. Your Business Plan includes 200GB ' +
```
Comment (lancewillett) [inline client/blocks/product-purchase-features-list/video-audio-posts.jsx]: Let's change this to include a space between the number and the measurement. `200 GB`

File: client/blocks/product-purchase-features-list/video-audio-posts.jsx [hunk 3]
```diff
@@ -19,8 +19,8 @@ import videoImage from 'assets/images/illustrations/video-hosting.svg';
 function getDescription( plan, translate ) {
 	if ( isWpComBusinessPlan( plan ) ) {
 		return translate(
-			'Enrich your posts and pages with video or audio. Upload as much media as you want, ' +
-				'directly to your site — the Business Plan has unlimited storage.'
+			'Enrich your site and boost engagement with video and audio. Your Business Plan includes 200 GB ' +
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features-list/video-audio-posts.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Enrich your site and boost engagement with video and audio. Your Business Plan includes 200GB of storage, giving you plenty of room for all types of media.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/my-plan)

File: client/blocks/product-purchase-features-list/video-audio-posts.jsx [hunk 4]
```diff
@@ -32,7 +32,7 @@ function getDescription( plan, translate ) {
 
 	if ( isWpComPremiumPlan( plan ) ) {
 		return translate(
-			'Enrich your posts and pages with video or audio. Upload up to 13GB of media directly to your site.'
+			'Enrich your posts and pages with video or audio. Upload up to 13 GB of media directly to your site.'
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features-list/video-audio-posts.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Pull request closed without merging**